### DISCUSSION TITLE
2.10.0

### DISCUSSION THREAD

**Discussion by TheLastProject** [CatimaLoyalty/Android#605] [Releases]
- New main screen layout
- Fix bottomsheet sizing issues when switching in and out of fullscreen

<hr /><em>This discussion was created from the release <a href='https://github.com/TheLastProject/Catima/releases/tag/v2.10.0'>2.10.0</a>.</em>

**Comment by JVariance** [comment]
The new layout is awesome 🙌🏼
Could you add black/white as color for the titlebar (separate titlebar and accent color)?

**Reply by TheLastProject** [reply]
Glad you like the new layout!

Implementing black/white as theme colour makes more sense. On that note, the main screen top bar shouldn't ignore the theming colour, that's a bug.

**Reply by JVariance** [reply]
I'm not sure if I explained my request adequately :D
So, I want to keep the accent color on buttons etc., but want the titlebar to not be affected by the accent color.
So it is black on dark theme and white on light theme.
![20211127_092937.jpg](https://user-images.githubusercontent.com/62245444/143674256-ecc4c36f-6b6d-4409-b1e6-8e828484e0de.jpg)

**Reply by TheLastProject** [reply]
Hmm. Well, I get what you mean but that's the kind of code I don't really want to introduce for one single person as it isn't good for an app to have too many options (it just confuses people). So sorry, not going to add that, a black/white theme and some fixes for when the title bar doesn't use the correct colour is the closest I'm okay with doing :x

**Reply by JVariance** [reply]
Okay. Maybe you could then add a darker gray as accent color?
